CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The grid.Builder API now allows users to specify options for intermediate
   containers, i.e. containers that don't have widgets, but represent rows and
   columns.
+- Line chart widget now allows `math.NaN` values to represent "no value" (values
```

Comment:
One more request in relation to the CLA - can you please explicitly confirm on this PR that I have your permission to merge this into the master branch in the next release?

Reply:
@mum4k has permission to merge this PR in master branch in the next release.